Submissions/@Aakibvah.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "MahmmadAakib Vahora",
+  "img": "https://avatars.githubusercontent.com/u/87711863?v=4",
+  "email": "aakibv290@gmail.com",
+  "links": {
+    "website": "https://aakibvah.github.io/",
+    "linkedin": "https://www.linkedin.com/in/mahmmadaakibvahora",
+    "github": "https://github.com/Aakibvah"
+  },
+  "jobTitle": "Web Developer [React and Node]",
+  "location": {
+    "city": "Toronto",
+    "state": "Ontario",
+    "country": "Canada]"
```

Comment:
sorry I think I missed this one - please remove the ] from Canada, as I do not believe this will show up on the map